src/core/CoolBoard.cpp

```diff
@@ -44,130 +44,130 @@ void CoolBoard::begin() {
+      this->coolBoardLed.write(WHITE);
       delay(5000);
-      coolBoardLed.write(0, 30, 0);
-      bValue = irene3000.readButton();
+      this->coolBoardLed.write(GREEN);
+      bValue = this->irene3000.readButton();
       if (bValue >= 65000) {
         bValue = 0;
       }
       while (bValue > 2000) {
-        bValue = irene3000.readButton();
+        bValue = this->irene3000.readButton();
         if (bValue >= 65000) {
           bValue = 8800;
         }
         delay(1000);
       }
-      coolBoardLed.write(0, 255, 0);
+      this->coolBoardLed.write(GREEN);
       INFO_LOG("Calibrating the pH probe");
       INFO_LOG("pH 7 calibration for 25 seconds");
       delay(10000);
-      irene3000.calibratepH7();
+      this->irene3000.calibratepH7();
       delay(15000);
-      irene3000.calibratepH7();
-      coolBoardLed.write(30, 0, 0);
-      bValue = irene3000.readButton();
+      this->irene3000.calibratepH7();
+      this->coolBoardLed.write(RED);
+      bValue = this->irene3000.readButton();
       if (bValue >= 65000) {
         bValue = 8800;
       }
       while (bValue > 2000) {
-        bValue = irene3000.readButton();
+        bValue = this->irene3000.readButton();
         if (bValue >= 65000) {
           bValue = 8800;
         }
         delay(1000);
       }
-      coolBoardLed.write(255, 0, 0);
+      this->coolBoardLed.write(RED);
       INFO_LOG("pH 4 calibration for 25 seconds");
       delay(10000);
-      irene3000.calibratepH4();
+      this->irene3000.calibratepH4();
       delay(15000);
-      irene3000.calibratepH4();
-      irene3000.saveParams();
-      coolBoardLed.write(30, 0, 30);
-      bValue = irene3000.readButton();
+      this->irene3000.calibratepH4();
+      this->irene3000.saveParams();
+      this->coolBoardLed.write(FUCHSIA);
+      bValue = this->irene3000.readButton();
       if (bValue >= 65000) {
         bValue = 8800;
       }
       while (bValue > 2000) {
-        bValue = irene3000.readButton();
+        bValue = this->irene3000.readButton();
         if (bValue >= 65000) {
           bValue = 8800;
         }
         delay(1000);
       }
     }
-    irene3000.printConf();
+    this->irene3000.printConf();
     delay(100);
   }
 
   if (externalSensorsActive) {
-    externalSensors.config();
-    externalSensors.begin();
+    this->externalSensors.config();
+    this->externalSensors.begin();
     delay(100);
-    externalSensors.printConf();
+    this->externalSensors.printConf();
   }
 
   this->connect();
   delay(100);
-  rtc.begin();
+  this->rtc.begin();
 
   if (this->sleepActive == 0) {
     this->sendConfig("CoolBoard", "/coolBoardConfig.json");
     this->sendConfig("CoolSensorsBoard", "/coolBoardSensorsConfig.json");
     this->sendConfig("CoolBoardActor", "/coolBoardActorConfig.json");
     this->sendConfig("rtc", "/rtcConfig.json");
-    this->sendConfig("led", "/coolBoardLedConfig.json");
+    this->sendConfig("led", "/this->coolBoardLedConfig.json");
```

Comment:
Search/Replace detected ! Please look at the diff more carefully next time. This is how bugs happen.